app/src/main/java/org/sagebionetworks/bridge/researchstack/CrfTaskFactory.java

```diff
@@ -285,6 +382,20 @@ private void fillCrfStartTaskStep(CrfStartTaskStep step, CrfStartTaskSurveyItem
     private CrfHeartRateCameraStep createHeartRateCameraStep(ActiveStepSurveyItem item) {
         CrfHeartRateCameraStep step = new CrfHeartRateCameraStep(item.identifier, item.title, item.text);
         fillCrfActiveStep(step, item);
+        if(item.identifier != null) {
+            step.stepIdentifier = item.identifier;
+        }
+        if(item.buttonType != null) {
```

Comment:
I think you added a new buttonType field to ActiveStepSurveryItem, which is part of ResearchStack.  We probably figure out a way to add it to the CRF project.